android/app/src/main/java/app/organicmaps/SplashActivity.java

```diff
@@ -196,4 +202,13 @@ public void processNavigation()
     startActivity(intent);
     finish();
   }
+
+  private boolean isManageSpaceActivity(Intent intent) {
+    var component = intent.getComponent();
+
+    if (!Intent.ACTION_VIEW.equals(intent.getAction())) return false;
+    if (component == null) return false;
+
+    return "app.organicmaps.ManageSpaceActivity".equals(component.getClassName());
```

Comment:
use BuildConfig.APPLICATION_ID?

Reply:
To make it work, "$applicationId" is also has to be used in AndroidManifest in all spots where this activity-alias is mentioned. Not sure it worth it.

Reply:
Having it concentrated in the Manifest is OK (makes it trivial to replace when needed).
Having the string app id spread over the codebase is less nice, seems like other places in the code use `BuildConfig.APPLICATION_ID` when needed